doc/tasks.rst

```diff
@@ -307,3 +307,19 @@ Luigi also does some metaclass logic so that
 if e.g. ``DailyReport(datetime.date(2012, 5, 10))`` is instantiated twice in the code,
 it will in fact result in the same object.
 See :ref:`Parameter-instance-caching` for more info
+
+
+Atomicity of Operations on Targets
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Atomicity is crucial when working with targets. In the following scenarios
+atomicity cannot be guaranteed.
+
+Moving of a :class:`~luigi.target.LocalTarget` across filesytems:
+
+.. code:: python
+
+    luigi.LocalTarget(path=src).move(dst)
```

Comment:
Yes, you're right. I addressed this the [next PR](https://github.com/spotify/luigi/pull/1877/commits/93f064d3bc0a7602b65a6af532a7f0f20f2c7dc0).